webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const Encore = require('@symfony/webpack-encore');
```

Comment:
This file should be in `tests/Application`. It is useless in plugin root.

Compare it wit old one in `tests/Application` and decide if we will adopt some changes or even replace old one by this new one (injected probably by Flex)